mlos_bench/mlos_bench/config/schemas/tunables/tunable-params-schema.json

```diff
@@ -13,6 +13,23 @@
                 }
             }
         },
+        "tunable_param_distribution": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": ["uniform", "normal", "beta"]
+                },
+                "params": {
+                    "type": "object",
+                    "items": {
+                        "type": "number"
+                    },
+                    "minItems": 1
```

Comment:
For auto complete purposes it might be good to include the types of parameters we expect here.
Let it be free form (e.g., allow others), but also include the known properties as of this time (e.g., "mu" and "sigma" for type == "normal", "alpha" and "beta" for "type" == "beta").

Since we know it's currently a fixed set of supported distributions I think this shouldn't be too hard, and also makes it easier for people to fill things in without having to refer to ConfigSpace source code (which also isn't obvious, so may want to additionally include a reference to https://automl.github.io/ConfigSpace/main/api/hyperparameters.html#distributions in the "description" field).

If you want I can cook up a PR for this.

Reply:
https://github.com/motus/MLOS/pull/9

Reply:
Since there were conflicts in this PR due to the previous merge, I've cherry picked the functionality from it into this PR. Thank you!